sktime/classification/kernel_based/_arsenal.py

```diff
@@ -22,10 +21,9 @@
 
 
 class Arsenal(BaseClassifier):
```

Comment:
Can we rename that to `ArsenalClassifier` so that we can distinguish it from its regressor counterpart? I think most classifiers now have Classifier in their name, it's just clearer and what people will expect coming from scikit-learn.

Reply:
No

Reply:
Ok